src/components/Sum.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
+const Sum = ({ a, b }) => {
+  const text = `Sum of ${a} and ${b} is ${a + b}`;
```

Comment:
no need to create a separate variable for this, just add to jsx in return statement